test-e2e/tag-in-image.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+oc tag --source=docker $IMAGE_PREPARE openshift/jenkins:2
```

Comment:
maybe a comment about how IMAGE_PREPARE is set/inherited from ci-operator

(and in general what this script/image is for)

Reply:
script comment explaining things pushed